createEnvironment.js

```diff
@@ -48,6 +48,7 @@ function createEnvironment({ baseEnvironment } = {}) {
         name: this.testPath,
         tags: transactionOptions.tags,
       });
+      this.global._transaction = this.transaction;
```

Comment:
Since we're exposing it should we remove the underscore?

```suggestion
      this.global.transaction = this.transaction;
```